ssr/src/page/token/transfer.rs

```diff
@@ -117,6 +118,79 @@ async fn transfer_token_to_user_principal(
     Ok(())
 }
 
+async fn transfer_ck_token_to_user_principal(
+    cans_wire: CanistersAuthWire,
+    destination_principal: Principal,
+    ledger_canister: Principal,
+    amount: TokenBalance,
+) -> Result<(), ServerFnError> {
+    let cans = cans_wire.canisters().unwrap();
+    // let user_id = user_id.to_owned();
+    // let user_principal = user_id.sender()?;
+    // let agent = cans.agent.get_agent().await;
+    // let user_principal = agent.get_principal()?;
+    // log::debug!("user_principal: {:?}", user_principal.to_string());
+    let sns_ledger = cans.sns_ledger(ledger_canister).await;
+    let res = sns_ledger
+        .icrc_1_transfer(TransferArg {
+            memo: Some(serde_bytes::ByteBuf::from(vec![0])),
+            amount: amount.clone().into(),
+            fee: None,
+            from_subaccount: None,
+            to: Account {
+                owner: destination_principal,
+                subaccount: None,
+            },
+            created_at_time: None,
+        })
+        .await
+        .unwrap();
```

Comment:
?

Reply:
I just noticed that these problems exist in `transfer_token_to_user_principal` as well, feel free to fix that function as well